core/controllers/cron.py

```diff
@@ -35,17 +35,27 @@
 MAX_JOBS_TO_REPORT_ON = 50
 
 
-def require_cron_or_superadmin(handler):
+def require_cron_or_superadmin(func):
     """Decorator to ensure that the handler is being called by cron or by a
     superadmin of the application.
+
+    Args:
+        func: The cron function to be decorated
```

Comment:
Need to specify type.

    func: function. The cron function toe be decorated.

Also, here and below, add full stop at the end of each sentence (after 'decorated', 'function' and 'URL').